File: Examples/CSharp/WorkingWithTaskLinks/CreateTaskLinks.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithTaskLinks
{
    internal class CreateTaskLinks
    {
        public static void Run()
        {
            //ExStart:CreateTaskLinks
            //ExFor: Project.TaskLinks
            //ExSummary: Shows how to create task links.
            // Create new project and add tasks
            var project = new Project();
            var pred = project.RootTask.Children.Add("Task 1");
            var succ = project.RootTask.Children.Add("Task 2");

            // Links tasks
            project.TaskLinks.Add(pred, succ);
            //ExEnd:CreateTaskLinks
        }
    }
}

File: Examples/VisualBasic/WorkingWithTaskLinks/CreateTaskLinks.vb

Namespace WorkingWithTaskLinks
    Public Class CreateTaskLinks
        Public Shared Sub Run()
            ' ExStart:CreateTaskLinks
            ' Create new project and add tasks
            Dim project1 As New Project()
            Dim pred As Task = project1.RootTask.Children.Add("Task 1")
            Dim succ As Task = project1.RootTask.Children.Add("Task 2")

            ' Link tasks
            Dim link As TaskLink = project1.TaskLinks.Add(pred, succ)
            ' ExEnd:CreateTaskLinks
        End Sub
    End Class
End Namespace

